modules/tinymce/src/core/test/ts/browser/fmt/FormatEmptyLineTest.ts

```diff
@@ -1,12 +1,17 @@
+import { Keys } from '@ephox/agar';
 import { context, describe, it } from '@ephox/bedrock-client';
-import { TinyAssertions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
+import { Type, Unicode } from '@ephox/katamari';
+import { TinyAssertions, TinyContentActions, TinyHooks, TinySelections, TinyUiActions } from '@ephox/wrap-mcagar';
 
 import Editor from 'tinymce/core/api/Editor';
 
 interface TestConfig {
   readonly selector: string;
   readonly selectorCount: number;
   readonly html: string;
+  readonly contentFormatRaw?: boolean;
```

Comment:
Instead of making this a boolean maybe just make this a string/`Content` type? That way it's clearer what it's used for in the test. e.g: `contentFormat: Content` and then you'd do `contentFormat: 'raw'` 🤷

Reply:
I think the other thing that's confusing me is that the content is passed via `html` here and then this uses `content` so they don't appear to be related. So another option would be to call this `rawHtml` instead which would probably be clearer than my previous suggestion.

Reply:
Renamed property to `rawHtml`